pkg/fuzzy/google_translate.go

```diff
@@ -126,36 +132,56 @@ func (g *GoogleTranslate) Translate(
 		Messages: make([]model.Message, 0, n),
 	}
 
-	for low := 0; low < n; low += googleTranslateRequestlimit {
-		high := low + googleTranslateRequestlimit
+	for low := 0; low < n; low += googleTranslateRequestLimit {
```

Comment:
Can you simplify it? Can you use the two constraints (limits) as a threshold for batching?